skill-tree /src/test/java/com/RDS/skilltree/TestContainerManager.java

```diff
@@ -0,0 +1,24 @@
+package com.RDS.skilltree;
+
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.MySQLContainer;
+
+abstract class TestContainerManager {
```

Comment:
why are we marking this class as abstract?

Reply:
As in this class we can have some methods not defined and to be defined by its child classes, this class had to be extended by the other classes and used.

Reply:
I don't see any abstract methods here.

Reply:
Yes they are no as of now